design-proposals/nic-hotplug/nic-hotplug.md

```diff
@@ -377,6 +390,118 @@ spec:
 The aforementioned update will trigger multus to start the CNI ADD flow for the
 network named `macvlan-conf-2`.
 
+### Unplug for pods
+Following #hotplug-for-pods example, to unplug an interface, update the pod to:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-case-03
+  annotations:
+    k8s.v1.cni.cncf.io/networks: macvlan-conf-2
+spec:
+  containers:
+  - name: pod-case-03
+    image: docker.io/centos/tools:latest
+    command:
+    - /sbin/init
+```
+
+The aforementioned update will trigger multus to start the CNI DEL flow for the
+network named `macvlan-conf-1`.
+
+## Backward Compatibility
+### Legacy VM's virt-launcher pods interface naming
+*Legacy VM - running VMs that were created prior to Kubevirt upgrade
+which run on top of previous version virt-launcher pod*
+
+Changing the virt-launcher pod interface name scheme breaks backward compatibility 
+in a way that legacy VMs won't be able to migrate, See the diagram below:
+![](upgrades-and-pod-iface-nameing-issue.png)
+
+1. The VM originally runs on top of virt-launcher pod from version v0.59.0
+2. Kubevirt upgrades to v0.60.0.
+3. The VM is migrated.
+4. The migration target pod is created from the new image (v0.60.0) with interface names in form of the new name-scheme - `7e0055a6880`.
+But the interface name in the migration domain XML is in form of the old name scheme - `tap1`.
+
+The proposed solution is having virt-controller to create the migration target pod with  
+`k8s.v1.cni.cncf.io/networks` annotation with same pod interface names as the migration 
+source pod annotation.
+
+The migration target pod interfaces names will then match the names in the 
+incoming migration domain XML and the migration process will start.
+
+In case the user migrates the VM again, same as before, the migration target 
+`pods k8s.v1.cni.cncf.io/networks` annotation value will have the same interface 
+names as in the migration source annotation.
+
+### Unplug interface of a legacy VM
+#### Story 1
+Running legacy VM with secondary networks, with the following networks in the spec:
+```yaml
+spec:
+networks:
+- name: blue-network
+  multus:
+  networkName: blue-net-br
+- name: red-network
+  multus:
+  networkName: red-net-br
+```
+The VM pod network-status annotation will look as follows:
+```yaml
+k8s.v1.cni.cncf.io/networks-status: '[
+    { "name": "blue-net-br", "interface": "net1", ...},
+    { "name": "red-net-br", "interface": "net2", ...},
+]'
+```
+
+Unplugging `net1` and `net2` should be blocked because it makes mapping between 
+the VMI networks names and the pod interfaces names impossible.
```

Comment:
This sentence is written from Multus' perspective (an implementation detail).
Please consider changing the explanation to a VM's perspective.